Strict mode issues

## Description  
Enable TypeScript's strict mode (tsconfig.json: "strict": true) and the following exceptions would be thrown.

 * igniteui-angular version: 10.2.x, 11.0.x
## Result
Error: node_modules/@infragistics/igniteui-angular/lib/services/csv/csv-exporter.d.ts:42:5 - error TS2416: Property 'onExportEnded' in type 'IgxCsvExporterService' is not
  Type 'EventEmitter<ICsvExportEndedEventArgs>' is not assignable to type 'EventEmitter<IBaseEventArgs>'.
    Types of property 'observers' are incompatible.
      Type 'Observer<ICsvExportEndedEventArgs>[]' is not assignable to type 'Observer<IBaseEventArgs>[]'.
        Type 'Observer<ICsvExportEndedEventArgs>' is not assignable to type 'Observer<IBaseEventArgs>'.
          Type 'IBaseEventArgs' is not assignable to type 'ICsvExportEndedEventArgs'.

       ~~~~~~~~~~~~~
node_modules/@infragistics/igniteui-angular/lib/services/excel/excel-exporter.d.ts:43:5 - error TS2416: Property 'onExportEnded' in type 'IgxExcelExporterService' is not
  Type 'EventEmitter<IExcelExportEndedEventArgs>' is not assignable to type 'EventEmitter<IBaseEventArgs>'.
    Types of property 'observers' are incompatible.
      Type 'Observer<IExcelExportEndedEventArgs>[]' is not assignable to type 'Observer<IBaseEventArgs>[]'.
        Type 'Observer<IExcelExportEndedEventArgs>' is not assignable to type 'Observer<IBaseEventArgs>'.
          Type 'IBaseEventArgs' is not assignable to type 'IExcelExportEndedEventArgs'.

43     onExportEnded: EventEmitter<IExcelExportEndedEventArgs>;

## Expected Result:
No exceptions to be thrown.

--- a/projects/igniteui-angular/src/lib/services/csv/csv-exporter.ts
+++ b/projects/igniteui-angular/src/lib/services/csv/csv-exporter.ts
@@ -6,7 +6,7 @@ import { CsvFileTypes, IgxCsvExporterOptions } from './csv-exporter-options';
 import { IBaseEventArgs } from '../../core/utils';
 
 export interface ICsvExportEndedEventArgs extends IBaseEventArgs {
-    csvData: string;
+    csvData?: string;
 }
 
 /**

--- a/projects/igniteui-angular/src/lib/services/excel/excel-exporter.ts
+++ b/projects/igniteui-angular/src/lib/services/excel/excel-exporter.ts
@@ -12,7 +12,7 @@ import { IBaseEventArgs } from '../../core/utils';
 import { WorksheetFile } from './excel-files';
 
 export interface IExcelExportEndedEventArgs extends IBaseEventArgs {
-    xlsx: JSZip;
+    xlsx?: JSZip;
 }
 
 /**

